doc/release-notes/release-notes-4.7.0-rc1.md

```diff
@@ -0,0 +1,538 @@
+Notable changes
+===============
+
+Mnemonic Recovery Phrases
+-------------------------
+
+The zcashd wallet has been modified to support BIP 39, which describes how to
+derive the wallet's HD seed from a mnemonic phrase.  The mnemonic phrase will
+be generated on load of the wallet, or the first time the wallet is unlocked,
+and is available via the `z_exportwallet` RPC call. All new addresses produced
+by the wallet are now derived from this seed using the HD wallet functionality
+described in ZIP 32 and ZIP 316. For users upgrading an existing Zcashd wallet,
+it is recommended that the wallet be backed up prior to upgrading to the 4.7.0
+Zcashd release.
+
+Following the upgrade to 4.7.0, Zcashd will require that the user confirm that
+they have backed up their new emergency recovery phrase, which may be obtained
+from the output of the `z_exportwallet` RPC call. This confirmation can be
+performed manually using the `zcashd-wallet-tool` utility that is supplied
+with this release (built or installed in the same directory as `zcashd`).
+The wallet will not allow the generation of new addresses until this
+confirmation has been performed. It is recommended that after this upgrade,
+that funds tied to preexisting addresses be migrated to newly generated
```

Comment:
```suggestion
funds tied to preexisting addresses be migrated to newly generated
```

Reply:
This can be fixed in the release notes updates before 4.7.0.